assets/app/view/user.rb

```diff
@@ -123,6 +125,18 @@ def render_notifications(checked = true)
       ])
     end
 
+    def render_desktop_notifications(checked = false)
+      h('div#settings__desktop_notifications', [
+        render_input(
+          'Allow Desktop Turn and Message Notifications while in game',
+          id: :desktop_notifications,
+          type: :checkbox,
+          attrs: { checked: checked },
+          on: { change: -> { Lib::Notification.ask_permission } },
```

Comment:
I could be wrong, but this appears to ask permission whether the user is checking or unchecking the option. Shouldn't it only ask if the box is checked?